src/ui/components/LedgerAdmin.js

```diff
@@ -71,12 +85,28 @@ export const LedgerAdmin = (): ReactNode => {
   const [nextIdentityName, setIdentityName] = useState<string>("");
   const [selectedId, setSelectedId] = useState<IdentityId | null>(null);
   const [promptString, setPromptString] = useState<string>("Add Identity:");
-  const [checkboxSelected, setCheckBoxSelected] = useState<boolean>(false);
   const accounts = useMemo(() => ledger.accounts().map((a) => a.identity), [
     ledger._latestTimestamp,
   ]);
   const accountsTableState = useTableState({data: accounts});
 
+  const isCheckedHash: Map<IdentityId, boolean> = useMemo(() => {
+    const isCheckedHash = new Map();
+    ledger.accounts().map(({identity, active}) => {
+      isCheckedHash.set(identity.id, active);
+    });
+    return isCheckedHash;
+  }, [ledger]);
+
+  const handleSingleCheck = useCallback(
+    (id: IdentityId, event: SyntheticInputEvent<HTMLInputElement>) => {
+      const checked = event.currentTarget.checked;
+      toggleIdentityActivation(id);
+
+      isCheckedHash.set(id, checked);
+    }
+  );
```

Comment:
This setup is kinda funky and bug-prone since there's two sources of truth for whether an account is active or not (the ledger itself + the `isCheckedHash` map).

Additionally, the way that isCheckedHash is using `useMemo` will cause the same logic to run many times unnecessarily. For example:
1. User loads page and `isCheckedHash` map is created and memoized based on the current ledger
2. User clicks a check box to activate an identity which sets the corresponding entry in `isCheckedHash` and also updates the ledger when it calls `toggleIdentityActivation`
3. Since the ledger has changed, the useMemo call for `isCheckedHash` runs again which iterates over all the ledger accounts and sets the same values that are already set, which will cause UI lag and choppiness as well as potential desynchronization of what's in the ledger and what the UI is showing.

The solution to this is just to not have an `isCheckedHash` value at all and instead use the ledger itself as the single source of truth for whether an account is active or not. Instead of doing `isCheckedHash.get(identityId)` for determining if a row is checked or not, you can just do `ledger.account(identityId)?.active` and in `handleSingleCheck` you just call the `toggleIdentityActivation` function and get rid of the `isCheckedHash.set(id, checked)` call. Keeps it much more simple and declarative instead of trying to copy state back and forth between ledger and `isCheckedHash`